pkg/provider/helm/values_test.go

```diff
@@ -313,6 +441,726 @@ func TestHelmValuesGenerator_GenerateValues_failure(t *testing.T) {
 	}
 }
 
+func TestGetNestedMap(t *testing.T) {
+	tests := []struct {
+		name   string
+		values map[string]any
+		key    string
+		want   map[string]any
+		exists bool
+	}{
+		{
```

Comment:
There is a third case - the key exists in the map but is not a map, so the type assertion fails -> panic!

Reply:
This should now be handled by the new `getOrCreateNestedMap` method and [this](https://github.com/cofide/cofidectl/blob/490a943475f120a5d717ed53227831589b46b1bc/pkg/provider/helm/values_test.go#L481) test case!